### DISCUSSION TITLE
Cannot see offline state in the app and amount of peers connected, no mobile data usage flow

### DISCUSSION THREAD

**Discussion by churik** [status-im/status-legacy#16664] [General]
# Problem

On 1.20 when the user was offline/or on the cellular network it was possible and very handy to see the special state on the home view, which opens an informational popup on tap (with status node and peer-to-peer info)

Currently, there is no sign for user if they go from WI-Fi network, that can be a reason for the overuse of cellular traffic.
The task is about creating some temporary solution to get info about the app state + number of connected peers, which can be useful for debug purposes.

No design is still available for this part.

On video from 1.20:
1) User switches to the mobile network, choose the option for using the app over cellular data, tap on the connection state on the home view
2) User goes offline, can see the change in the connection state, can tap on it, and see there are no peers connected

https://user-images.githubusercontent.com/4557972/209182821-487ce856-f3fc-4f12-8ee0-5f16a8970270.mp4

## Implementation
Ugly, but understandable placeholders for described flows.

## Acceptance Criteria

For all these options as there are no designs we need to consider that  they must look ugly (from the design team) and not something that looks like a potential implementation (see https://github.com/status-im/status-mobile/pull/14616 for more context):
- can see when I'm offline
- can see when I'm switching to the cellular network (when mobile usage data is switched on or off)
- can see the number of peers I'm connected to

**Comment by rasom** [comment]
well if you say it should look ugly....

**Comment by churik** [comment]
I'll re-open this one, as it requires design anyway, it is missing in design

**Comment by churik** [comment]
cc @xAlisher @pedro-et
